AutomatedTesting/Gem/PythonTests/scripting/scripting_utils/scripting_constants.py

```diff
@@ -73,6 +73,8 @@
 WAIT_TIME_3 = 3
 WAIT_TIME_5 = 5
 VARIABLE_TYPES = ["Boolean", "Color", "EntityId", "Number", "String", "Transform", "Vector2", "Vector3", "Vector4"]
```

Comment:
Make a comment note here that this is deprecated and will be replaced as this conversion takes place.